recipes/muparser/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "2.2.5" %}
+
+package:
+  name: muparser
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/beltoforion/muparser/archive/v{{ version }}.tar.gz 
+  md5: 02dae671aa5ad955fdcbcd3fee313fb7
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - gcc  # [linux]
```

Comment:
I am not familiar with `muparser`, but unless you need a Fortran compiler or `libgomp` we discourage the use of the default channel `gcc` package.